BRANDING.MD

```diff
@@ -0,0 +1,39 @@
+## Knative Branding Guidelines
+
+As a general rule, Knative follows the [Linux Foundation trademark
+usage](https://www.linuxfoundation.org/trademark-usage/) rules. As a high-level
+summary, the following is made for easy reference.
```

Comment:
This last bit is confusing -- if there's a conflict between the text here and the Linux Foundation usage rules, which set of rules governs?

(Note that this could happen by Linux Foundation updating their rules.)

Reply:
@evankanderson updated it - see if that addresses your concen

Reply:
Do we need to say this bit even? If the summary as written covers the majority of the uses cases, and anything outside that would go to the TC, shouldn't that be enough? Referencing another guideline that we don't have any say in could just be confusing.